setup.py

```diff
@@ -11,8 +11,8 @@
 
 PY_VER = sys.version_info
 
-if PY_VER < (3, 5):
-    raise RuntimeError("aioredis doesn't support Python version prior 3.5")
+if PY_VER < (3, 5, 3):
+    raise RuntimeError("aioredis doesn't support Python version prior 3.5.3")
```

Comment:
Why 3.5.3?
There is not strict requirement on `.3` version.
I've just run the tests on Python 3.5.2
Can you please clarify why exactly `3.5.3`?

Reply:
I've just picked it up from the readme: https://github.com/aio-libs/aioredis/blob/master/README.rst#requirements

Reply:
Oh, I really don't remember what was the reason for exactly 3.5.3.
We, probably, can stick with `3.5.3`

Reply:
IIRC 3.5.3 changes `get_event_loop` so that when called from inside a task it always returns the event loop in which the task is running, even if it isn't the default event loop for the thread. I don't know if that is relevant to aioredis, but I believe it's the reason aiohttp now requires 3.5.3.